packages/devtools/lib/src/charts/flame_chart_canvas.dart

```diff
@@ -395,13 +331,16 @@ class FlameChartRow {
   final int index;
 }
 
-class FlameChartNode {
+typedef DisplayTextProvider<T> = String Function(T);
```

Comment:
https://dart.dev/guides/language/effective-dart/design#prefer-inline-function-types-over-typedefs

The typedef is shorter than the name - is the name adding enough value?